packages/form-generator/package.json

```diff
@@ -15,9 +15,14 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
+    "post:compile": "npm run copy:dependencies",
+    "copy:dependencies": "mkdir -p node_modules && cp -R ../../node_modules/@graphql-codegen node_modules/",
```

Comment:
This is due to
https://github.com/npm/cli/issues/3466
and 
https://github.com/npm/rfcs/issues/287